src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1493,6 +1495,8 @@ void goto_instrument_parse_optionst::help()
     // NOLINTNEXTLINE(whitespace/line_length)
     " --reachable-call-graph       show graph of function calls potentially reachable from main function\n"
     " --class-hierarchy            show class hierarchy\n"
+    // NOLINTNEXTLINE(whitespace/line_length)
+    " --show-threaded              show instructions that may be executed by more than one thread\n"
```

Comment:
This is just documented and undocumented option, right?

Reply:
Yes, it's only documenting a previously undocumented option.